docs/blocks/custom-fluids.md

```diff
@@ -0,0 +1,851 @@
+---
```

Comment:
Just as a general overall in case I missed it, make all identifiers -> `wiki:blockname`